account_invoice_triple_discount/readme/DESCRIPTION.rst

```diff
@@ -1 +1,3 @@
 This module allows to have three successive discounts on each invoice line.
+
+If the module `account_invoice_fixed_discount` is installed and a fixed discount is set, it will take priority over anything set in the triple discount fields.
```

Comment:
I think that for end users, having the field hidden by a glue module is a better UX.